cadc-util/src/main/java/ca/nrc/cadc/auth/SignedToken.java

```diff
@@ -248,6 +248,25 @@ private static StringBuilder getContent(SignedToken token) {
         return sb;
     }
 
+    /**
+     * Checks whether a token is a CADC access token or not
```

Comment:
"a CADC access token" -> SignedToken

also, `Base64.decodeString(String)` exists and could be used instead of `new String`... it's the same code but intended for this use case

should be `catch(IllegalArgumentException ignore)`


Q. Why does this handle plain text and base64-encoded test? And why is the condition different in those two cases? Aren't all SignedToken values in headers base64 encoded?

Reply:
Fixed. Can't remember why plain text. I've removed it.